src/facade/dragonfly_connection.cc

```diff
@@ -1016,10 +1029,13 @@ void Connection::DispatchFiber(util::FiberSocketBase* peer) {
     queue_backpressure_->ec.notify();
   }
 
-  cc_->conn_closing = true;
+  if (builder->GetError())
```

Comment:
why this change?

Reply:
Will remove, I eventually kept the return below ShouldEndDispatchFiber. Otherwise breaking from the loop does not always equal closing the connection,(migration request)